README.md

```diff
@@ -1,45 +1,55 @@
 # Faasm Covid Experiment
```

Comment:
Would it make sense to have this `README` be structured in the same way of [`experiment-mpi`](https://github.com/faasm/experiment-mpi)?

Namely just have two sub-sections: _Running on Faasm_ and _Running natively_